package.json

```diff
@@ -17,56 +17,58 @@
     "type": "module",
     "devDependencies": {
         "@capacitor/assets": "^2.0.4",
-        "@capacitor/cli": "^5.2.2",
-        "@ianvs/prettier-plugin-sort-imports": "^4.1.0",
-        "@playwright/test": "^1.36.2",
-        "@types/node": "^18.17.1",
-        "@typescript-eslint/eslint-plugin": "^6.2.0",
-        "@typescript-eslint/parser": "^6.2.0",
-        "autoprefixer": "^10.4.14",
+        "@capacitor/cli": "^5.5.1",
+        "@ianvs/prettier-plugin-sort-imports": "^4.1.1",
+        "@playwright/test": "^1.39.0",
+        "@types/node": "^20.8.9",
+        "@typescript-eslint/eslint-plugin": "^6.9.0",
+        "@typescript-eslint/parser": "^6.9.0",
+        "autoprefixer": "^10.4.16",
         "esbuild": "^0.14.54",
-        "eslint": "^8.46.0",
+        "eslint": "^8.52.0",
         "eslint-import-resolver-typescript": "2.7.1",
         "eslint-plugin-import": "2.27.5",
         "eslint-plugin-prettier": "4.2.1",
         "eslint-plugin-solid": "0.13.0",
-        "postcss": "^8.4.27",
-        "prettier": "^3.0.2",
-        "prettier-plugin-tailwindcss": "^0.5.3",
-        "solid-start-node": "^0.2.29",
-        "tailwindcss": "^3.3.3",
-        "typescript": "^5.1.6",
-        "vite": "^4.4.7",
+        "postcss": "^8.4.31",
+        "prettier": "^3.0.3",
+        "prettier-plugin-tailwindcss": "^0.5.6",
+        "solid-start-node": "^0.3.8",
+        "tailwindcss": "^3.3.5",
+        "typescript": "^5.2.2",
+        "vite": "^4.5.0",
         "vite-plugin-pwa": "^0.14.7",
         "vite-plugin-wasm": "^3.2.2",
         "workbox-window": "^6.6.0"
     },
     "dependencies": {
-        "@capacitor/android": "^5.2.2",
+        "@capacitor-community/barcode-scanner": "npm:@mutinywallet/barcode-scanner@5.0.0-beta.3",
```

Comment:
This has been deprecated for this: https://github.com/capawesome-team/capacitor-mlkit/tree/main/packages/barcode-scanning